### PR TITLE
fix bug on Decimal serialize with null values

### PR DESCRIPTION
**Pull request by jordic**
Fixes a bug on Decimal serialitzation when the value is null (None)

### CODE REVIEW

File: typesystem/fields.py [hunk 1]
```diff
@@ -310,7 +310,7 @@ class Decimal(Number):
     numeric_type = decimal.Decimal
 
     def serialize(self, obj: typing.Any) -> typing.Any:
-        return float(obj)
+        return float(obj) if obj else None
```
Comment (tomchristie) [inline typesystem/fields.py]: That doeesn't look right to me for the `0` case.

Comment (tomchristie) [reply typesystem/fields.py]: Probably `return None if obj is None else float(obj)`, right?

Comment (jordic) [reply typesystem/fields.py]: Thirth time on the same week with the same mistake... haha, seems like a complex relation with python conditional booleans :)

**Comment by tomchristie** [comment]
Thanks!

**Pull request merged**